components/sharing-dialog/i18n/en.pot

```diff
@@ -149,8 +149,8 @@ msgstr "Can view and edit"
 msgid "View and edit"
 msgstr "View and edit"
 
-msgid "Sharing & Access: {{objectName}}"
-msgstr "Sharing & Access: {{objectName}}"
+msgid "Sharing & Access: {{- objectName}}"
+msgstr "Sharing & Access: {{- objectName}}"
```

Comment:
This seems like an artifact from some other change?

Reply:
Yeah I think so - probably just didn't get committed with the change that created it

Reply:
Is it okay to leave in?